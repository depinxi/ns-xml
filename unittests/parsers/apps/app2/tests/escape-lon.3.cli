--argument-b "Escaping long option name" "\--switch-a" 