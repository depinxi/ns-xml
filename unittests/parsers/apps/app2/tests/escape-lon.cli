-a "\--argument-b" "Escaping long option name" 