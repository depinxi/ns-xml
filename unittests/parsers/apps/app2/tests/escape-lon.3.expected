CLI: "--argument-b", "Escaping long option name", "\--switch-a"
Value count: 1
Values: "--switch-a"
Error count: 0
Subcommand: 
switchA=False
argumentB=Escaping long option name
multiargC=
